tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -147,16 +148,18 @@ sub run_test {
 
     }
 
+    set_var("SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH", 0);    #turn off the flag in case of affecting other tests
```

Comment:
What is this used for ?

Reply:
sriov test share a function with virtual network in lib, but there is a bit difference which has to be flagged by a variable. In other words, they need a runtime variable to flag which test called the function. Although sriov tests are separated as a standalone test, running along with virtual network tests are supported as well.

Reply:
But if you need to flag which test calls a function, a variable like SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH is not a good choice, because it also depends test execution order. If other test run before pci passthrough, then detecting SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH becomes useless. The correct way is to use "caller", for exmaple,

sub myFunc{
  my ($package, $filename, $line) = caller;
  print $filename;
}

This way enables you to differentiate which module or file calls shared function.